Sources/SpeziViews/Views/Text/MarkdownView.swift

```diff
@@ -95,14 +96,18 @@ public struct MarkdownView: View {
     /// Parses the incoming markdown and handles the view's error state management.
     /// - Parameters:
     ///   - markdown: A `Data` instance containing the markdown file in an utf8 representation.
-    ///
+    ///   ```swift
+    ///   // Example
+    ///   Data(" # This is a markdown example
+    ///    *This should be italiced* and **this bolded**").utf8
+    ///    ```
     /// - Returns: Parsed Markdown as an `AttributedString`
     @MainActor private func parse(markdown: Data) -> AttributedString {
         state = .processing
         
         guard let markdownString = try? AttributedString(
                 markdown: markdown,
-                options: .init(interpretedSyntax: .inlineOnlyPreservingWhitespace)
+                options: .init(interpretedSyntax: .inlineOnly)
```

Comment:
What is the reason for this change? Wouldn't it make sense to switch to `.full`?